apps/frontpage/app/sitemap-all/sitemap.xml

```diff
@@ -0,0 +1,11 @@
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```

Comment:
I don't understand why this isn't `app/sitemap-all.xml`?

If you did that, everything would be under the same directory (the site root). Thus, there'd be no need to generate duplicate sitemaps for addons and recipes, and the docs one could live at `app/docs/sitemap-all.xml`.

Reply:
You're right. The trick to make this happen is to put `sitemap-all.tsx` in the public folder since `sitemap.xl` is a special naming. I just can't put it next to `sitemap.xml`.

Reply:
This is now fixed.